### DISCUSSION TITLE
460800 baud rate on raspberry pi

### DISCUSSION THREAD

**Discussion by rprj** [openthread/openthread#7837] [Q&A] [answered]
Hi everyone,

I tried to increase the baud rate to 460800 on my raspberry pi, but now the otbr-agent keeps resetting again and again. Does anyone know why this happens?

![image](https://user-images.githubusercontent.com/108110547/175456943-b6379770-d63b-41e1-ae67-f0dcc97fa244.png)

I changed the baud rate in /etc/default/otbr-agent

![image](https://user-images.githubusercontent.com/108110547/175457445-e774cbff-86bf-423b-acd6-126603f8f0e5.png)

**Comment by jwhui** [comment, marked as answer]
I suspect your RCP is not configured for the same baud rate. Can you verify that your RCP configured with the same baud rate?
